corehq/toggles.py

```diff
@@ -1805,6 +1805,14 @@ def _commtrackify(domain_name, toggle_is_enabled):
 )
 
 
+USER_API_USE_ES_BACKEND = StaticToggle(
+    'user_api_use_es_backend',
+    'Use new ES backend for User API.',
+    TAG_PRODUCT,
+    [NAMESPACE_DOMAIN, NAMESPACE_USER],
+)
```

Comment:
> The plan is to test this in production against real data under this flag, and then release it with the old method under an inverse flag (USER_API_USE_ES_COUCH), and then delete the old method altogether.

What's the idea behind using the inverse flag at all?  A burndown list of domains to remove manually?  Will it be a predictably random toggle with a frequency you decrease over time to roll out incrementally?  A safeguard in case something goes wrong and you need to re-enable the old way for a domain in a hurry?

Reply:
Yeah a safeguard in case it goes wrong and support needs to quickly unblock a particular domain. I did the same thing with the group API, and had no complaints and will remove that escape hatch in a day or two and remove the old group API backend. I expect it to go similarly for users